### ISSUE TITLE
git tag failing with "fatal: too many params" on some tagAnnotation values

### ISSUE THREAD

**Issue by piotrb** [release-it/release-it#576]
Hi,

I'm including the whole changelog in the tag annotation value, which should normally be ok as far as git is concerned, but it seems like because of how the data is passed to the command it causes it to fail.

To achieve this I'm using the following in my package.json

```json
{
  "release-it": {
    "git": {
      "tagAnnotation": "Release ${version}\n\n${changelog}"
    }
}
```

This works fine with simpler changelogs, but I happen to be releasing something that has ~60 commits in it and I guess some of those commits have quotes in them ... and it causing it fail:

```
? Tag (1.5.0)? Yes
fatal: too many params
  release-it:shell { command:
  release-it:shell    `git tag --annotate --message="Release 1.5.0\n\n**redacted**"  1.5.0`,
  release-it:shell   options: {},
  release-it:shell   code: 128,
  release-it:shell   stdout: '',
  release-it:shell   stderr: 'fatal: too many params\n' } +7s
```

What I'm seeing in the --message value though is some values like:

```
tweak the "delta" display in a pod to not consider
Update Terminal Sessions to "Task Stopped" when
```

etc .. 

whis is probably breaking the quoting .. 

It seems like the value of --message should be passed through a shell quoting library to make sure its escaping unsafe characters.

**Comment by piotrb** [comment]
It's seem the change would be needed around here:

https://github.com/release-it/release-it/blob/master/lib/plugin/git/Git.js#L133

**Comment by webpro** [comment]
I just released `12.5.0-next` which should contain a fix for this (`npm install release-it@next`). Can you please try this? Thanks!

**Comment by piotrb** [comment]
Yup that looks like it fixed it .. made a new release that changed my changelog format back and used some quotes in the commit entry ... worked like a charm:

```
$ git show 1.5.2
tag 1.5.2
Tagger: Piotr Banasik <piotr@jane.app>
Date:   Mon Dec 2 09:56:09 2019 -0800

Release 1.5.2

* Switch back to full changelog tag annotation that "supports" quotes (26bd7a24)
```
